nimterop/cimport.nim

```diff
@@ -715,10 +713,6 @@ macro c2nImport*(filename: static string, recurse: static bool = false, dynlib:
   var
     nimout = &"const {header} = \"{fullpath}\"\n\n" & readFile(npath)
 
-  nimout = nimout.
```

Comment:
This is not necessary anymore?

Reply:
Not sure if anyone uses c2nim with nimterop. Will consider if someone complains.